packages/connectivity/src/scp-cf/destination/destination-accessor-types.ts

```diff
@@ -6,15 +6,15 @@ import type {
 import type { DestinationSelectionStrategy } from './destination-selection-strategies';
 
 /**
- * @internal
+ * PLEASE REVIEW.
```

Comment:
@FrankEssenberger I have an idea, but it is safer if you do it 😊

Reply:
```suggestion
Collection of all destinations from the provider and subscriber account.
The used [[DestinationSelectionStrategy]] will decide which destination is selected in the end.
```